### PR TITLE
Add plugin dashboard page sample

### PR DESCRIPTION
**Pull request by acicovic**
## Description
This PR adds a sample plugin dashboard page.

## Motivation and context
We want to add a plugin dashboard page from which the Traffic Boost feature (and possibly other features) will be available.

## How has this been tested?
Manually tested that the React app renders, as seen in the screenshot.

## Screenshots
image

## Summary by CodeRabbit

- **New Features**
	- Introduced a new dashboard page for the Parsely plugin in the WordPress admin interface.
	- Added a user-friendly layout with a welcoming message on the dashboard.

- **Bug Fixes**
	- Streamlined imports by removing unused classes, enhancing plugin performance.

- **Documentation**
	- Improved comments for better clarity on the new dashboard functionality.

- **Chores**
	- Updated build configuration to support new TypeScript files and dashboard page styling.

### CODE REVIEW

**Comment by acicovic** [comment]
The SonarCloud fail can be ignored, it's about code duplication that will be insignificant after merge.

**Comment by acicovic** [comment]
@vaurdan, I think we should review/merge this in order to avoid having too many PRs based on other PRs.

**Review by vaurdan — APPROVED** [review]
Let's merge!

**Pull request merged**
